packages/clerk-js/src/ui/components/SignIn/SignInFactorOne.tsx

```diff
@@ -83,21 +85,27 @@ function SignInFactorOneInternal(): JSX.Element {
   const [isPasswordPwned, setIsPasswordPwned] = React.useState(false);
 
   React.useEffect(() => {
+    if (__internal_setActiveInProgress) {
```

Comment:
Here's the behavior - right during `setActive`, this component re-renders with an empty sign-in status: 


https://github.com/user-attachments/assets/67e5b142-6603-451e-99ac-2c02262c0f15